package-lock.json

```diff
@@ -4,6 +4,33 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "@sindresorhus/is": {
```

Comment:
your `package-lock.json` has changes that are not required by this project. you will need to clean this up or remove from your next commit